### PR TITLE
feat(ci): /build_test PR command 

### PR DESCRIPTION
**Pull request by mwangggg**
# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
* [x] Read the [contributing guidelines](https://github.com/cryostatio/cryostat/blob/main/CONTRIBUTING.md)
* [x] Linked a relevant issue which this PR resolves
* [x] Linked any other relevant issues, PR's, or documentation, if any
* [x] Resolved all conflicts, if any
* [x] Rebased your branch PR on top of the latest upstream `main` branch
* [x] Attached at least one of the following labels to the PR: `[chore, ci, docs, feat, fix, test]`
* [x] [Signed all commits using a GPG signature](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#gpg-commit-signature-verification)

**To recreate commits with GPG signature** `git fetch upstream && git rebase --force --gpg-sign upstream/main`
_______________________________________________

Fixes: #1053

### CODE REVIEW

**Comment by mwangggg** [comment]
demo: https://github.com/mwangggg/cryostat-web/pull/2

File: .github/workflows/pr-ci.yml [hunk 1]
```diff
@@ -1,33 +1,76 @@
...
+              repo,
+              comment_id: context.payload.comment.id,
+              content: "+1",
+            });  
+
+  checkout-branch: 
+    runs-on: ubuntu-latest
+    needs: [check-before-build]
+    permissions:
+      contents: read
```
Comment (tthvo) [inline .github/workflows/pr-ci.yml:52]: I think `pull_requests: read` is enough? Can you check?

**Review by tthvo — COMMENTED** [review]
Looks good! Just a few comments.

File: .github/workflows/pr-ci.yml (hunk 1 above)
Comment (tthvo) [reply .github/workflows/pr-ci.yml]: Nice thanks for checking!

File: .github/workflows/pr-ci.yml [hunk 2]
```diff
@@ -1,33 +1,74 @@
...
+      if: ${{ contains(github.event.issue.labels.*.name, 'needs-triage') }}
+      run: exit 1
+    - name: Show warning if permission is denied
+      if: |
+        !(github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
+        && (!contains(github.event.issue.labels.*.name, 'safe-to-test') || github.event.issue.user.name != github.event.comment.user.name)
+      uses: thollander/actions-comment-pull-request@v2
+      with:
+        message: |-
+          You do not have permission to run the /build_test command. Please ask @cryostatio/reviewers
```
Comment (tthvo) [inline .github/workflows/pr-ci.yml:29]: Any instance of `/build_test` should be updated to `/build_image`?

**Review by tthvo — APPROVED** [review]
LGTM^^

File: .github/workflows/pr-ci.yml [hunk 3]
```diff
@@ -1,33 +1,74 @@
...
-      - cryostat-v[0-9]+.[0-9]+
+      - created
 
 jobs:
-  get-pom-properties:
+  check-before-build:
     runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_image')
```
Comment (andrewazores) [inline .github/workflows/pr-ci.yml:17]: I'd rather have the comment/command be consistent across repos, even if the naming isn't spot-on for each particular repo, just so reviewers don't need to remember which specific command works for which repo.

**Comment by andrewazores** [comment]
rebase please

**Review by tthvo — APPROVED** [review]

**Pull request merged**